xlb/operator/boundary_condition/bc_zouhe.py

```diff
@@ -210,11 +230,11 @@ def get_normal_vectors(
             if wp.static(_d == 3):
                 for l in range(_q):
                     if missing_mask[l] == wp.uint8(1) and wp.abs(_c[0, l]) + wp.abs(_c[1, l]) + wp.abs(_c[2, l]) == 1:
-                        return -_u_vec(_c_float[0, l], _c_float[1, l], _c_float[2, l])
+                        return -_d_vec(_c_float[0, l], _c_float[1, l], _c_float[2, l])
```

Comment:
I was personally confused first that this is a velocity vector and I was checking why we need this velocity here that's why I changed it to _d_vec. Both I fine I changed it because of my confusion.

Reply:
i.e., since normal is not a velocity. I think _d_vec probably makes more sense and more general for both.

Reply:
Either way we should make it consistant.

Reply:
Not a good name since it confused me. But ok.